.env

```diff
@@ -6,14 +6,11 @@ TARGET_DIR=$WORK_DIR/target
 TOOLS_DIR=$TARGET_DIR/tools
 
 # Dev dir where to clone keycloak
-# KC_TARGET_BRANCH=main
-# KC_TARGET_BRANCH=DEPLOY-20240725
-KC_TARGET_BRANCH=DEPLOY-BSK-DEMO-86
-KC_OID4VCI="keycloak_${KC_TARGET_BRANCH}"
+KEYCLOAK_OID4VCI_TARBALL=$TARGET_DIR/keycloak-26.0.6.tar.gz
```

Comment:
It is definitely more efficient to download built archives, but it will still be needed to build our own keycloak with custom changes not released. Can you make these options co-exist?